index.d.ts

```diff
@@ -310,6 +330,27 @@ export interface StringifyOptions {
 	```
 	*/
 	readonly skipEmptyString?: boolean;
+
+	/**
+	Adds the fragment identifier to the URL.
+
+	@default false
+
+	@example
+	```
+	import queryString = require('query-string');
+
+	queryString.stringifyUrl({
+		url: 'https://foo.bar',
+		query: {top: 'foo'},
+		fragmentIdentifier: 'bar'
+	}, {
+		parseFragmentIdentifier: true
```

Comment:
This option name makes no sense for **stringify**Url. Should it even need an option here?